### ISSUE TITLE
Integrate RVM authorities.

### ISSUE THREAD

**Issue by redlibrarian** [ualbertalib/discovery#901]
**Is your feature request related to a problem? Please describe.**
Items are catalogued in English and/or French in a number of different ways, and cataloguing practices have changed over the years. This presents a problem for French language searchers as a search for a French language subject does not return all possible results, and vice versa
**Describe the solution you'd like**
As a discovery user, I would like to be able to search for subjects in French and English and get similar results. To achieve this, a potential solution would be to integrate the [RVM Authorities](https://rvmweb.bibl.ulaval.ca/en/a-propos) purchased by the Libraries a number of years ago into SOLR a synonym file
 **Describe alternatives you've considered**
No alternatives have been considered at this point
 **Additional context**
This feature needs much more discussion and consultation with Bib Services colleagues on which thesauri should be implemented, and an assessment of the risk it poses to providing relvant search results

**Comment by seanluyk** [comment]
@pgwillia @weiweishi  - comment updated with more details

**Comment by seanluyk** [comment]
Adding more details from Brian:

We currently have the complete authority file for R.V.M. terms consisting of French subject headings and their equivalent L.C. subject headings. In the authority records, the L.C.S.H. equivalencies, and C.S.H. equivalencies if applicable, are included, coded with an indicator to identify their source and sometimes with the control number of the equivalent term. In Workflows, we currently have the L.C.S.H. authority file for terms that are used in our bibliographic records, the complete R.V.M. authority file, and the complete C.S.H. authority file.

Currently, our cataloguing practice is to describe all resources in English and to provide L.C.S.H. terms for all resources and to additionally provide R.V.M. terms for all resources that are either in French or are for B.S.J. Formerly, we catalogued resources for B.S.J. in French and assigned R.V.M. terms for them.

Ideally, I think that we would want users to be able to do a subject search in either English or French and return all resources about that subject, whether the controlled vocabulary term in the bibliographic record is from L.C.S.H. or from R.V.M.

This specific project would be limited to topical subjects and not names. This is an ideal time to restart this project, however, because L.A.C. is moving their English-language name authority file into the L.C. authority file. They will maintain a separate French-language name authority file, and name headings that they create in the L.C. name authority file will include their equivalent French-language form. This should allow us to extend bilingual subject searching to names (L.C. authority records generally will already have a French form of name if it has been used by the entity, so this enhanced capability will be of benefit mostly for personal names that have different forms in English and French).

At this time, there would be no consideration of changing local cataloguing policy to include R.V.M. terms as required.

I am including four truncated representative authority records: the first two are R.V.M. subjects, and the last two are names, from L.C. and L.A.C. respectively. They all show the equivalencies recorded in the fields 7XX, with the second indicator (and subfield $2 if necessary) indicating the source of the term.

016  \\$a0504H9388 
150  \\$aArbres
360  \\$iaux classes, ordres, espèces, etc., par ex.$aÉpinette ; Orme ; Pin
450  \\$aLigneux (Plantes)
450  \\$aDendrologie
450  \\$aArbres forestiers
450  \\$aEssences forestières
450  \\$aEssences de lumière
450  \\$aArbres indigènes
550  \\$wg$aPlantes ligneuses
550  \\$wg$aProduits de pépinières
550  \\$aArboriculture
550  \\$aBois d'œuvre
750  \2$aTrees
750  \0$aTrees$0(DLC)sh 85137241
750  \7$atrees$2aat
750  \7$aArbres$2ram
750  \7$aDendrology$2rvm

016  \\$a0509F5616
043  \\$an-cn---
045  \\$aw1w1 
151  \\$aCanada$xHistoire$y1812, Guerre de
450  \\$aGuerre de 1812
451  \\$aQuébec (Province)$xHistoire$y1812, Guerre de
451  \\$aOntario$xHistoire$y1812, Guerre de
551  \\$aÉtats-Unis$xHistoire$y1812, Guerre de
680  \\$iSous cette vedette de nom géographique, on trouve les documents sur les événements qui ont eu lieu au Canada. Pour les documents généraux sur cette guerre et les documents sur les événements qui ont eu lieu aux États-Unis, événements auxquels des Canadiens ont participé, on ajoute la vedette$aÉtats-Unis--Histoire--1812, Guerre de
751  \5$aCanada$xHistory$yWar of 1812
751  \0$aOntario$xHistory$yWar of 1812

010  \\$ano2018073500
016  \\$a0056B9451E
046  \\$s1982$t1988$2edtf
110  2\$aCanadian Coalition Against Media Pornography
377  \\$aeng
410  2\$aCCAMP
410  2\$aCoalition canadienne contre la pornographie dans les médias
710  27$aCoalition canadienne contre la pornographie dans les médias$2lacnaf$0(CaOONL)0056B9451F‏

016  \\$a0006J6860E
046  \\$f1926
100  0\$aElizabeth$bII,$cQueen of Great Britain,$d1926-

**Comment by seanluyk** [comment]
Out of scope pending discovery strategy recommendation - unfortunately won't get to this one before the freeze

**Comment by seanluyk** [comment]
For discussion at next grooming meeting @pgwillia to investigate. Has broader implications, e.g. decolonizing description, changed headings, etc.

**Comment by pgwillia** [comment]
@seanluyk What would you like me to investigate specifically?  What question do you want me to answer?

**Comment by seanluyk** [comment]
@pgwillia we're looking at how authority data such as RVM could be utilized in Blacklight. For example, if someone searches for an item and gets a match based on an English subject heading, items with equivalent French subject headings are returned.  The initial idea when I investigated this was the use of a synonym file (link to writeup in comment above), but given the potential size of this authority file (likely exceeding 1 million term pairs) @weiweishi thought this might break.

**Comment by weiweishi** [comment]
I'm leaning towards to make this as a future improvement - looking at examples from other institutions, I feel this is doable with some Solr investigation - but feel this is lower priority comparing to our other solr related tasks? (I've also created a Solr-related tag to identify these tickets, and I feel once we've finished some of the UI related tasks like the record view, we should focus more on these solr related tasks 
 to expand our knowledge in Solr )

**Comment by seanluyk** [comment]
@weiweishi I agree  - with something of this scale we want to get it right, and it's dependent a bit on our improved understanding of SOLR. Just adding this link back here again, because when I had looked into it before the use of synonym files was much less extensive, but something we should definitely be looking into for more straightforward things (e.g. computer language names, abbreviations, musical keys, etc.): https://docs.google.com/document/d/1TfFKyZru7umb5pV85bLURH6GPVZVKpWF1XKc4PCdbGs/edit?usp=sharing
